# OFFSET event-triggered distributed estimation simulation config

max_time: 20
dt: 0.1

monte_carlo_sims: True
num_mc_sims: 100

delta_values: [0.5,1.0,1.5,2.0]
tau_values: [2,3,5,7]
msg_drop_prob_values: [0.0,0.2,0.5,0.8]

use_adaptive_tau: True
fixed_rng: 999

agents:
    conn_struct_type: "tree"
    conns: [[9],[9],[10],[10],[11],[11],[12],[12],
            [1,2,13],[3,4,13],[5,6,14],[7,8,14],[9,10,15],[11,12,15],[13,14,16],
            [15,17,18],[16,19,20],[16,21,22],[17,23,24],[17,25,26],[18,27,28],
            [18,29,30],[19],[19],[20],[20],[21],[21],[22],[22]]
    dynamics: "lin_ncv"
    sensors:
        lin_abs_pos: [13,14,17,18]
        lin_rel_range: "all"

baseline:
    dynamics: "lin_ncv"
    sensors:
        lin_abs_pos: [13,14,17,18]
        lin_rel_range: "all"
